lib/src/test/java/io/ably/lib/test/realtime/ConnectionManagerTest.java

```diff
@@ -388,4 +389,32 @@ public void run() {
 			fail("init0: Unexpected exception instantiating library");
 		}
 	}
+
+	/**
+	 * Connect and then verify that the connection manager has the default value for connectionStateTtl;
+	 */
+	@Test
+	public void connection_details_has_ttl() {
+		try {
+			ClientOptions opts = createOptions(testVars.keys[0].keyStr);
+			final AblyRealtime ably = new AblyRealtime(opts);
+			ably.connection.on(ConnectionEvent.connected, new ConnectionStateListener() {
+				@Override
+				public void onConnectionStateChanged(ConnectionStateChange state) {
+					assertEquals(1, 1);
```

Comment:
What's the point of this?

Reply:
Why is this still here?

Reply:
@paddybyers @mattheworiordan Removed https://github.com/ably/ably-java/pull/389/commits/26c18f67a27d90d292ad53e3fed881f645bde190